assets/js/dependencies/async_loader.js

```diff
@@ -138,16 +137,16 @@
           }
           var base = $('base').attr('href');
           if(url.indexOf('?')>-1){
-            url = url + '&viewer_pane=1&RS_SHARED_SECRET=' + urlParam("RS_SHARED_SECRET")+"&Rstudio_port=" + urlParam("Rstudio_port");
+            url = url + '&rstudio_layout=1&viewer_pane=1&RS_SHARED_SECRET=' + urlParam("RS_SHARED_SECRET")+"&Rstudio_port=" + urlParam("Rstudio_port");
```

Comment:
While we are cleaning, you actually can use ajax like this:
```
$.ajax({
type: 'GET',
data: {
    rstudio_layout: '1',
    viewer_pane: '1',
    RS_SHARED_SECRET: urlParam("RS_SHARED_SECRET")
  }
})
```
and it will convert them into urlParam automatically